common/models/acl.js

```diff
@@ -392,6 +393,7 @@ module.exports = function(ACL) {
     self.resolveRelatedModels();
     var roleModel = self.roleModel;
 
+    context.registry = ACL.registry;
     if (!(context instanceof AccessContext)) {
       context = new AccessContext(context);
     }
```

Comment:
in order to fetch user models from within access-context.js, we need a handle on the app local registry whereas nothing currently enables us to do this. 
I propose to build AccessContext objects with the app registry attached so we can get it using this.registry inside AccessContext methods definitions